R/tm_g_waterfall.R

```diff
@@ -265,14 +265,15 @@ ui_g_waterfall <- function(id, ...) {
         value = a$gap_point_val
       )
     ),
-    forms = get_rcode_ui(ns("rcode")),
+    forms = teal.widgets::verbatim_popup_ui(ns("rcode"), "Show R code"),
     pre_output = a$pre_output,
     post_output = a$post_output
   )
 }
 
 srv_g_waterfall <- function(id,
```

Comment:
The validation note is not correct:
![Screenshot from 2022-09-07 16-06-14](https://user-images.githubusercontent.com/86970066/188899041-0811ee5e-2fda-4e00-9e68-34fd1fd4a600.png)